chai.js

```diff
@@ -0,0 +1,7 @@
+import chai from 'chai';
+import chaiEnzyme from 'chai-enzyme';
+
+chai.use(chaiEnzyme());
```

Comment:
Note sure what this file is for

Reply:
Adding chai allowed me to use the 'expect' statements w/in Enzyme. I know I *shouldn't* have to add chai, but when I added it - the error messages went away.